cromwell-helm/values.yaml

```diff
@@ -1,8 +1,8 @@
 config:
   gcsBucket: "SUPPLIED BY LEO"
   gcsProject: "SUPPLIED BY LEO"
-  gcsRegion: us-central1 # TODO: SUPPLY VIA LEO (BW-924)
-  papiv2EndpointUrl: https://lifesciences.googleapis.com/ # TODO: SUPPLY VIA LEO... or hard-code into Cromwell? (BW-924)
+  gcsRegion: "SUPPLIED BY LEO"
```

Comment:
Is Leo already supplying this and we're just not using it? Wondering if it's easier to leave this as-is.

Reply:
Oh we are using it, I already modified leo to supply it, testing now.